### DISCUSSION TITLE
Can’t create guild?

### DISCUSSION THREAD

**Discussion by AnthonyPaulO** [chromiecraft/chromiecraft#5324] [General]
Hello everyone! I'm trying to create a guild for my family, we're all playing together, and I tried having them sign the guild charter and they click and nothing happens. Am I missing something? Thanks in advance!

**Comment by 55Honey** [comment]
Hello @AnthonyPaulO 
Thanks for reaching out.

Feel free to join our discord for even more and better user interaction: https://discord.gg/chromiecraft

How many signatures do you have already? Note that you don't need a full roll of 10.

**Reply by AnthonyPaulO** [reply]
I don't have any signatures. It says it requires four but I've been unable to get two of my boys to sign it, clicking sign doesn't seem to do anything. Thanks!

**Reply by 55Honey** [reply]
An account can only sign once, no matter how many characters it has. Do you use different accounts for each signature?

**Reply by AnthonyPaulO** [reply]
Yes, each of my boys have their own accounts. From your response I'm getting the impression that I shouldn't be experiencing this, so I'll try again later tonight; perhaps it was server lag? I know that sometimes this happens when I click on a guard and ask for directions to a profession trainer or whatever, and when I click sometimes there's no response, no matter how many times I click. Then I exit out, move my character a little and try again, and then it works. Perhaps it was something along those lines, but I tried moving my character around a few times and requesting signatures a few times and it still didn't work. I'll post an update later on.

**Reply by AnthonyPaulO** [reply]
Hello everyone! I managed to get it to work. It turns out that if you get a charter you need to get it signed and registered that same session; if you log out, it's no longer valid and you have to get rid of it and buy another. I did that and it worked. Thanks for your help!

**Reply by 55Honey** [reply]
That sounds like a bug to me. Feel free to bring it up in an issue on our bugtracker here: https://github.com/chromiecraft/chromiecraft/issues/new/choose
I'm glad you solved it!
